g2zproxy

```diff
@@ -159,14 +159,17 @@ class G2ZProxy(object):
                 # Get data from graphite API
                 res = urllib2.urlopen(url)
                 data = yaml.load(res)
-            except:
-                # if error, process next metric
+            except Exception as inst:
+                # if error, process next metric and print exception
+		print type(inst)
+		print inst.args
+		print inst 
```

Comment:
This part looks really not good. It's better to write exception to `log.error('%s', e)`, and you can format it better, or just print `inst`. Also, why did you call it `inst` and not an `e`?